Cargo.toml

```diff
@@ -185,7 +185,7 @@ datafusion-execution = { git = "https://github.com/spiceai/datafusion.git", rev
 datafusion-expr = { git = "https://github.com/spiceai/datafusion.git", rev = "a898971e2c2652abf924d9d60e4d9d0a12620cd9" }      # spiceai-45
 
 datafusion-federation = { git = "https://github.com/spiceai/datafusion-federation.git", rev = "1c814422cfbdeb7f7bd2399e3369656710413aa8" }                      # spiceai-45
-datafusion-table-providers = { git = "https://github.com/datafusion-contrib/datafusion-table-providers.git", rev = "15d2c7ce1e071eef86733f0d2971cab779203874" } # spiceai
+datafusion-table-providers = { git = "https://github.com/datafusion-contrib/datafusion-table-providers.git", rev = "d45f16cb8cec4009179a6e0e5979c6af15c8ad64" } # phillip/250511-app-name
```

Comment:
Update after https://github.com/datafusion-contrib/datafusion-table-providers/pull/335 is merged